curriculum/challenges/english/blocks/workshop-greeting-card/67333082ab35e414eb4ea70d.md

```diff
@@ -113,5 +114,4 @@ body {
 
 --fcc-editable-region--
 
-
```

Comment:
here there is still an empty line between the end of the code block and the editable region